test/OrchardCore.Tests/Apis/GraphQL/ContentItemsFieldTypeTests.cs

```diff
@@ -205,10 +206,13 @@ public async Task ShouldFilterByAliasIndexRegardlessOfInputFieldCase(string fiel
             await session.SaveAsync(ci);
             await session.SaveChangesAsync();
 
-            var type = new ContentItemsFieldType("Animal", new Schema(services), Options.Create(new GraphQLContentOptions()), Options.Create(new GraphQLSettings { DefaultNumberOfResults = 10 }));
+            var type = new ContentItemsFieldType("Animal",
+                new Schema(services),
+                Options.Create(new GraphQLContentOptions()),
+                Options.Create(new GraphQLSettings { DefaultNumberOfResults = 10 }));
 
             context.Arguments["where"] = new ArgumentValue(JObject.Parse(string.Concat("{ ", fieldName, ": { name: \"doug\" } }")), ArgumentSource.Variable);
-            var dogs = await ((LockedAsyncFieldResolver<IEnumerable<ContentItem>>)type.Resolver).Resolve(context);
+            var dogs = (await ((LockedAsyncFieldResolver<IEnumerable<ContentItem>>)type.Resolver).ResolveAsync(context)) as IEnumerable<ContentItem>;
```

Comment:
It's unfortunate that all results need to be cast. Is there a way to change the service return type? Or can results not be IEnumerable?

Reply:
Yes, I checked all references to that type. Direct calls to 'ResolveAsync' only exist in unit tests, mostly just to validate results